aviary/subsystems/propulsion/test/test_turboprop_model.py

```diff
@@ -185,25 +157,85 @@ def test_case_3(self):
         results = self.get_results(point_names)
         assert_near_equal(results, truth_vals)
 
-    def test_case_4(self):
+    def test_case_2(self):
         # test case using GASP-derived engine deck and default HS prop model.
         filename = get_path('models/engines/turboprop_1120hp.deck')
         test_points = [(0, 0, 0), (0, 0, 1), (.6, 25000, 1)]
         point_names = ['idle', 'SLS', 'TOC']
         truth_vals = [(112, 37.7, -195.8), (1120, 136.3, -644), (1742.5, 21.3, -839.7)]
 
-        self.prepare_model(filename, test_points, True)
+        self.prepare_model(filename, test_points)
+
+        num_nodes = len(test_points)
 
         self.prob.set_val(Aircraft.Engine.PROPELLER_DIAMETER, 10.5, units="ft")
         self.prob.set_val(Aircraft.Engine.PROPELLER_ACTIVITY_FACTOR,
                           114.0, units="unitless")
         self.prob.set_val(
             Aircraft.Engine.PROPELLER_INTEGRATED_LIFT_COEFFICIENT, 0.5, units="unitless")
+        self.prob.set_val(Dynamic.Mission.PROPELLER_TIP_SPEED,
+                          800.*np.ones(num_nodes), units="ft/s")
```

Comment:
This should be a variable based on the test point

Reply:
I'm not quite following, can you reword?

Reply:
the propeller_tip_speed should be a dynamic variable that changes with RPM, here (and I think elsewhere) it is treated as a constant (`800.*np.ones(num_nodes)`)

Reply:
🤷 I just renamed the existing test w/o changing the inputs. While it's possible for RPM to change, does implementing that actually impact this unittest? You'll get different output values, but it won't getting any different code coverage than the fixed speed prop the test originally had.